CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,21 @@
 
 import Foundation
 
-print("Hello, World!")
+//------------step2 아이스크림 그리기 --------------
+//아이스크림 부분을 그리는 함수
+func drawIceCream() {
+    for _ in 1...8 {
+        print("***********")
+    }
+}
+
+//막대기(손잡이) 부분을 그리는 함수
+func drawiceCreamBar() {
```

Comment:
사소하지만.. 놓치신 것 같아 코멘트 남겨봅니다 +_+
```suggestion
func drawIceCreamBar() {
```

Reply:
또치님 확인 감사합니다 ଘ(੭ˊᵕˋ)੭* ੈ✩‧₊˚
다음 PR때는 조금 더 꼼꼼하게 체크하겠습니다!